docs/usage/guides/visual/window-customization.md

```diff
@@ -85,7 +82,7 @@ document
   .addEventListener('click', () => appWindow.minimize())
 document
   .getElementById('titlebar-maximize')
-  .addEventListener('click', () => appWindow.maximize())
+  .addEventListener('click', () => appWindow.isMaximized() ? appWindow.unmaximmize() : appWindow.maximize())
```

Comment:
isMaximized is async, so needs an await here

Reply:
nice, will update it